gm8emulator/src/game.rs

```diff
@@ -216,8 +217,9 @@ pub struct Game {
 
     pub play_type: PlayType,
     pub stored_events: VecDeque<replay::Event>,
-    pub frame_limiter: bool, // whether to limit FPS of gameplay by room_speed
-    pub frame_limit_at: usize, // on which frame to start limiting FPS
+    pub frame_limiter: bool,   // whether to limit FPS of gameplay by room_speed
+    pub frame_limit_at: usize,   // on which frame to start limiting FPS
```

Comment:
I mean, discard these changes in lines, please.
```suggestion
    pub frame_limiter: bool, // whether to limit FPS of gameplay by room_speed
    pub frame_limit_at: usize, // on which frame to start limiting FPS
```